appendix/privacy.rst

```diff
@@ -91,17 +91,17 @@ GeoCalendar
 GeoIP
    No user information is stored or cached on geo.zammad.com.
 
-   One of Zammad’s security features is to track user sessions
-   based on the user’s browser and country of origin.
+   One of Zammad's security features is to track user sessions
+   based on the user's browser and country of origin.
    Suspicious login activity from a different browser or country may trigger Zammad
    to dispatch an alert email to the affected user.
    The GeoIP service is used to associate IP addresses to a geographic origin.
 
 Geolocation
-   Since Zammad’s geolocation service relies on Google’s
-   `Geocoding API <https://developers.google.com/maps/documentation/geocoding/policies>`_,
-   its use is subject to the `Google Privacy Policy <https://policies.google.com/privacy>`_.
+   Zammad's geolocation service relies on OpenStreetMap (OSM) unless you turned
+   it off. If you provide an address (or parts of an address) in an user
```

Comment:
`an user` => `a user`

Consonat form due to the pronounciation of the word, even though it starts with a vowel.